src/themes/material/assets/mat.css

```diff
@@ -198,3 +393,76 @@ h5 {
     font-variant: small-caps
 }
 
+.header-overrides {
+    background-color: #fafafa;
+    color: #616161 !important;
+    height: 90px !important;
```

Comment:
Is there a conflict forcing us to flag these as `!important`?
Otherwise we risk changing the header colors for anyone with a regular override.

This applies to some of the directives below as well like the `.brand-logo` class